### ISSUE TITLE
Restrict too long variable names

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#288]
# Rule request

## Thesis

I am suggesting a complete opposite rule to `TooShortNameViolation`.
Whenever user decides to use names like this:

```python
this_is_clearly_a_very_long_for_a_single_variable_ist_it = 12
```

We should warn him that this name is too long.
This rule must be configurable. I am not quite sure about the maximum length right now.
Maybe it should be `29`?

## Reasoning

Too long names are unreadable. It is better to use shorter alternative.
Long names also indicate that this variable is too complex, maybe it may require some documentation.

**Comment by Casva** [comment]
I could take this up!
